WikidataOrg.hooks.php

```diff
@@ -2,6 +2,7 @@
 
 namespace WikidataOrg;
 
+use Html;
 use OutputPage;
 use Skin;
 use Wikibase\Repo\WikibaseRepo;
```

Comment:
While not technically required, you should still add `QuickTemplate` and `SkinTemplate` here.